boards/modalai/voxl2/target/voxl-px4-hitl

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+CONFIG_FILE="/etc/modalai/voxl-px4-hitl.conf"
+
+DAEMON_MODE=DISABLE
+
+# Try to source configuration variables from a file.
+# This will override anything already set in the environment.
+if [ -f "$CONFIG_FILE" ]; then
+    echo "[INFO] Reading from $CONFIG_FILE"
+    source $CONFIG_FILE
+else
+    echo "[INFO] $CONFIG_FILE not found, using defaults"
+fi
+
+# Make sure that the SLPI DSP test signature is there otherwise px4 cannot run
+# on the DSP
+if /bin/ls /usr/lib/rfsa/adsp/testsig-*.so &> /dev/null; then
+    /bin/echo "Found DSP signature file"
+else
+    /bin/echo "[WARNING] Could not find DSP signature file"
+    # Look for the DSP signature generation script
+    if [ -f /share/modalai/qrb5165-slpi-test-sig/generate-test-sig.sh ]; then
+        /bin/echo "[INFO] Attempting to generate the DSP signature file"
+        # Automatically generate the test signature so that px4 can run on SLPI DSP
+        /share/modalai/qrb5165-slpi-test-sig/generate-test-sig.sh
+    else
+        /bin/echo "[ERROR] Could not find the DSP signature file generation script"
+        exit 0
+    fi
+fi
+
+print_usage() {
+    echo -e "\nUsage: voxl-px4-hitl"
+    echo "                [-c delete configuration file and exit]"
+    echo "                [-d start px4 in daemon mode]"
+    echo "                [-h (show help)]"
+
+    exit 1
+}
+
+while getopts "cdhz" flag
+do
+    case "${flag}" in
+        c) 
+            echo "[INFO] Wiping old config file"
+            if [ -f "$CONFIG_FILE" ]; then
+                rm -rf ${CONFIG_FILE}
+            fi
+            exit 0
+            ;;
+        d) 
+            echo "[INFO] Enabling daemon mode"
+            DAEMON_MODE=ENABLE
+            ;;
+        h) 
+            print_usage
+            ;;
+        *) 
+            print_usage
+            ;;
+    esac
+done
```

Comment:
Are these options going to change now? You'll have VIO vs GPS?

Reply:
Since we are using 1.14.0 beta and not 1.13 or below, VIO vs GPS is handled internally in the firmware by the EKF2_GPS_CTRL and EKF2_EV_CTRL parameters which are both enabled - since the 1.14 they are gunning for easy swap between gps and vio, we dont need to worry about parameters like ekf2_aid_mask or flags that will differentiate the two.